packit_service/service/events.py

```diff
@@ -697,18 +697,33 @@ def __init__(
     ):
         super().__init__(project_url=project_url)
         self.action = action
+        # issue_id = ID of the specific comment
         self.issue_id = issue_id
```

Comment:
so how about renaming it to issue_comment_id? :)

Reply:
it's used in database, damn... will try to do something about it with rebase

i'm more for merging, and refactor + fixes if there is some problem (it's GitLab and it's special :upside_down_face:)

Reply:
should be resolved